source/localizable/routing/specifying-a-routes-model.md

```diff
@@ -23,7 +23,7 @@ export default Ember.Route.extend({
 ```
 
 Typically, the `model` hook should return an [Ember Data](../../models/) record,
-but it can also return any [promise](https://www.promisejs.org/) object (Ember
```

Comment:
But it's not a `Promise` object as in the `Promise` class, it's an object that's also a promise. I don't think I've cleared anything up xD